molgenis/molgenis-emx2
# feat(directory): biobank services as a separate entity

What are the main changes you did:
- Removed the capabilities column from the biobanks table
- Added a services column to the biobanks table
- Created a new services table with the column provided the document in https://github.com/molgenis/GCC/issues/554
- Added some dummy services data for both biobanks
- Removed the capabilities ontology table

how to test:
- Goto the preview and check the directory-demo (or create a new schema using the biobank_directory template) and see that it includes a services table which is linked to the biobanks table

todo:
- [ ] updated docs in case of new feature => Datamanager Manual needs to be adjusted
- [nvt] added/updated testplan to include a test for this fix, including ref to bug using # notation
- If first round of review is OK:
- [ ] Add services table also to the staging area template
- [ ] Adjust the Directory app as it's not working properly anymore (#3770)
